docs/usage/interacting-with-node.md

```diff
@@ -1,11 +1,13 @@
 ---
 id: api-calls-postman
 order: 1
-title: API calls via Postman
+title: Rest API
 category: 3. Using the Centrifuge Node
 ---
 
-Once the Centrifuge node is up and running you are able to start submitting invoices and tokenize these documents via simple API calls. For this, you can either use Postman (see [Recommended tools](/docs/getting-started/tools) or use your standard command-line interface. If you use Postman, make sure you have imported the most recent [Centrifuge Postman Collection](https://www.getpostman.com/collections/828cc6af5bc56642ba42).
+Once the Centrifuge node is up and running you are able to start submitting invoices and tokenize these documents via the Rest API. Please refer to the [Node API](https://centrifuge-os-node-api-2.api-docs.io/0.0.5/) documentation for a complete list of endpoints.
+
+In the section below we show how you can interact with the API through Postman (see [Recommended tools](/docs/getting-started/tools). If you use Postman, can import the [Centrifuge Postman Collection](https://www.getpostman.com/collections/828cc6af5bc56642ba42).
```

Comment:
I wouldn't say **can** impart the PM collection. It is important that they have the latest, so the old wording "make sure you have imported the most recent" or similar is probably better